daemon/api.go

```diff
@@ -661,6 +661,7 @@ func postSnap(c *Command, r *http.Request, user *auth.UserState) Response {
 
 	chg := newChange(state, inst.Action+"-snap", msg, tsets)
 	chg.Set("snap-names", []string{inst.snap})
+	chg.Set("api-data", map[string]string{"snap-name": inst.snap})
```

Comment:
Thanks, I addressed this wart now, please let me know if that looks sane or if it needs further tweaking.